addons/l10n_id/models/account_move.py

```diff
@@ -0,0 +1,47 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.tools.misc import formatLang
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _compute_tax_totals(self):
+        """ OVERRIDE
+
+        For invoices based on ID company as of January 2025, there is a separate tax base computation for nun-luxury goods.
+        Tax base is supposed to be 11/12 of original while other tax bases should remain the same.
+
+        To avoid affecting calculation of total amount, set tax_group_amount to 0 but the formatted_tax_group_amount to
+        the DPP value
+        """
+        super()._compute_tax_totals()
+        dpp_label = "DPP Other Value (11 / 12)"
+        for move in self.filtered(lambda m: m.is_sale_document()):
+            non_luxury_tax_group_id = self.env.ref('l10n_id.l10n_id_tax_group_non_luxury_goods', raise_if_not_found=False).id
+            if not (move.invoice_date and move.invoice_date < fields.Date.to_date('2025-01-01')):
```

Comment:
This is easier to understand imo.
```suggestion
            if move.invoice_date and move.invoice_date >= fields.Date.to_date('2025-01-01'):
```
or maybe (to avoid one level of indentation)
```suggestion
            if not move.invoice_date or move.invoice_date < fields.Date.to_date('2025-01-01'):
                continue
            # The rest of the code
```